### PR TITLE
Creates Fed ETL Migration Guide to Aggregator

### PR DESCRIPTION
**Pull request by bstuder99**

### CODE REVIEW

File: install-and-configure/install/multi-cluster/federated-etl/federated-etl-migration-guide.md
Comment (patrick-tavares) [inline install-and-configure/install/multi-cluster/federated-etl/federated-etl-migration-guide.md:1]: [install/multi-cluster/federated-etl/federated-etl-migration-guide.md](https://github.com/kubecost/docs/pull/908/files/1a5930ddad5bdbf20580252ee2308fcdabdca13c#diff-d1115599461d84d55fc3e81f517fe4fb4699b116b4c0bdf22b316806e95cc644)

Comment (patrick-tavares) [inline install-and-configure/install/multi-cluster/federated-etl/federated-etl-migration-guide.md:1]: Line 1: Federated ETK: Should be ETL
Line 3: ...migrate from Kubecost Fedrate ETL: should be Federated ETL
Lines120 and 150: The helm upgrade command makes no mention of -f <values.yaml file> option for the values files that were likely updated as part of the migration process.

Comment (mclean0328) [reply install-and-configure/install/multi-cluster/federated-etl/federated-etl-migration-guide.md:1]: Agree with all comments.  For Lines120 and 150, we can change the command to the below command.

helm upgrade --install "kubecost" --repo https://kubecost.github.io/cost-analyzer/ cost-analyzer --namespace kubecost -f values.yaml

Comment (bstuder99) [reply install-and-configure/install/multi-cluster/federated-etl/federated-etl-migration-guide.md:1]: Good catches Patrick, I've issued corrections for all your comments

**Review by mclean0328 — APPROVED** [review]
This all looks good to me!

**Pull request merged**